app-web/gatsby/onCreateNode.js

```diff
@@ -236,17 +236,18 @@ module.exports = ({ node, actions, getNode, getNodes }) => {
       }
     });
 
-    let labels = [];
+    let labels = {};
     // assert the shape of labels in frontmatter
-    if (Object.prototype.hasOwnProperty(node.frontmatter, 'labels')) {
-      if (isArray(node.frontmatter.labels) && node.frontmatter.every(isString)) {
+    if (Object.prototype.hasOwnProperty.call(node.frontmatter, 'labels')) {
+      if (isObject(node.frontmatter.labels) && !isArray(node.frontmatter.labels)) {
```

Comment:
isObject(['aaa', 'bbb', 'ccc']) will return true... that's why I add isArrary here

Reply:
use isPlainObject instead.